### DISCUSSION TITLE
Future migration path

### DISCUSSION THREAD

**Discussion by skttl** [umbraco/Umbraco-CMS#13638] [Features and ideas]
In v11 the migrations from 8 to 9 was removed, as "[the only migration path expected to work is to update one major at a time](https://github.com/umbraco/Announcements/issues/1)".

How is this going to work going forward, if I want to be in LTS versions only, and upgrade from 10 to 13? Do I have to go through 11 and 12 first?

I think there should be migrations in place for going straight from one LTS to another at leat.

**Comment by ronaldbarendse** [comment]
It probably all depends on how many migrations there will be and how complex they are: you can currently still upgrade from 9.4 to the latest version (11.0.0 as of this writing).

The major reason for removing/obsoleting the migrations mentioned in that announcement was because there were a lot of migrations required to go from v7 to v8 (related to content variations) and there wasn't a supported way to migrate v7 directly to v10 anyway. It also massively simplified the migration plan (compare [9.4.3](https://github.com/umbraco/Umbraco-CMS/blob/release-9.4.3/src/Umbraco.Infrastructure/Migrations/Upgrade/UmbracoPlan.cs) against [11.0.0](https://github.com/umbraco/Umbraco-CMS/blob/release-11.0.0/src/Umbraco.Infrastructure/Migrations/Upgrade/UmbracoPlan.cs)), so it was mainly done to clean the codebase up and make the migration plan more maintainable (i.e. reduce bugs in the migration path).
